packages/vscode-extension/src/debug/teamsfxDebugProvider.ts

```diff
@@ -149,9 +148,16 @@ export class TeamsfxDebugProvider implements vscode.DebugConfigurationProvider {
       }
       debugConfiguration.url = result;
 
+      VsCodeLogInstance.info(sideloadingDisplayMessages.title(debugConfiguration.teamsfxHub!));
+      VsCodeLogInstance.outputChannel.appendLine("");
       VsCodeLogInstance.outputChannel.appendLine(
-        util.format(launchUrlMessage, debugConfiguration.teamsfxHub, url)
+        sideloadingDisplayMessages.sideloadingUrlMessage(
+          debugConfiguration.teamsfxHub!,
+          debugConfiguration.url
+        )
       );
+      VsCodeLogInstance.outputChannel.appendLine("");
+      VsCodeLogInstance.outputChannel.appendLine(sideloadingDisplayMessages.hotReloadingMessage);
```

Comment:
Will user confuse if it's a bot project?
And, where to tell when to re-f5 due to manifest change?

Reply:
Good questions. Updated to `The app supports hot reloading. If you have any code changes within the project, the app will be reloaded.`. Do not directly tell re-f5 is needed if manifest changed.